app/renderer/components/styles/global.js

```diff
@@ -106,6 +106,7 @@ const globalStyles = {
     navbarBraveButtonMarginLeft: '80px',
     navbarLeftMarginDarwin: '76px',
     sideBarWidth: '190px',
+    aboutPageDetailsPageWidth: '704px',
```

Comment:
this variable name is not very clear... Can we name it something more appropriate, like `aboutExtensionDetailWidth`?